src/netmd-interface.ts

```diff
@@ -1019,6 +1055,9 @@ export class MDSession {
         if (!this.hexSessionKey) {
             throw new Error(`Call init first!`);
         }
+        try {
+            await this.md.syncTOC();
+        } catch (ex) {}
```

Comment:
This try-catch is probably not needed anymore since the exception is now handled into the `syncTOC` method